src/main/scala/it/pagopa/interop/purposeprocess/api/impl/RiskAnalysisValidation.scala

```diff
@@ -20,23 +21,39 @@ object RiskAnalysisValidation {
 
   /** Validate a Process risk analysis form and returns the same in the Management format
     * @param form Risk Analysis Form
+    * @param kind The Tenant Kind
     * @return Validated risk analysis
     */
-  def validate(form: RiskAnalysisForm): ValidationResult[RiskAnalysisFormSeed] = {
+  def validate(form: RiskAnalysisForm)(kind: TenantKind): ValidationResult[RiskAnalysisFormSeed] = {
+    RiskAnalysisService.riskAnalysisForms
+      .get(kind)
+      .fold[ValidationResult[RiskAnalysisFormSeed]](UnexpectedTenantKind(kind).invalidNec)(
+        validateLatestVersion(_, kind)(form)
+      )
+  }
+
+  /** Validate a Process risk analysis form and returns the same in the Management format
+    * @param versions Versions for this Tenant Kind
+    * @param form Risk Analysis Form
+    * @return Validated risk analysis
+    */
+  private def validateLatestVersion(versions: Map[String, RiskAnalysisFormConfig], tenantkind: TenantKind)(
+    form: RiskAnalysisForm
+  ): ValidationResult[RiskAnalysisFormSeed] = {
 
     val sanitizedForm = form.copy(answers = form.answers.filter(_._2.nonEmpty))
 
     val validationRules: ValidationResult[List[ValidationEntry]] =
-      RiskAnalysisService.riskAnalysisForms
-        .get(sanitizedForm.version)
-        .fold[ValidationResult[List[ValidationEntry]]](UnexpectedTemplateVersion(sanitizedForm.version).invalidNec)(
-          configsToRules(_).validNec
+      versions.lastOption
```

Comment:
`Map` is not an ordered collection and `lastOption` may return unexpected result.
We need to use `maxByOption`